pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go

```diff
@@ -2061,7 +2061,8 @@ func TestBuildUpstreamClusterTLSContext(t *testing.T) {
 												},
 											},
 										},
-										ResourceApiVersion: core.ApiVersion_V3,
+										ResourceApiVersion:  core.ApiVersion_V3,
+										InitialFetchTimeout: durationpb.New(time.Second * 0),
```

Comment:
This means a bad DR/Gateway config can block a pod indefinitely?

Reply:
https://github.com/istio/istio/blob/f149180608e8e91ca7e686a0f7ffc01142f2a03d/security/pkg/nodeagent/cache/secretcache.go#L399 - We error out if file does not exist. So it should cover this case?